### ISSUE TITLE
Freeform text field in need creation UI

### ISSUE THREAD

**Issue by sarahwicks** [fosterful/scheduler#181]
User Story:

As a Social Worker, I need a way to communicate specific/unique needs of children coming into care so that Volunteers can evaluate whether they are willing and able to accept the request.

Dependencies:

- Is blocked by #180 

Examples:

- The child(ren) is ill
- The child(ren) has head lice.
- The child(ren) has a special need, e.g., is autistic or deaf.
- The child(ren) is recovering from surgery.

Acceptance Criteria:

- [ ] A user creating a need has the ability to add notes for each child included in the request.
- [x] The notes entered by the user creating the need are visible to volunteers who are viewing the need.

**Comment by haseebelahi** [comment]
Hi @sarahwicks, @nvick! 
I'm trying to get involved and this one also looks like a good first issue for me which I can tackle, should I pick this one up?

**Comment by benjaminwood** [comment]
@haseebelahi I believe this one is actually dependent on another issue (#180). I'm removing the "good first issue" label for now.

@sarahwicks can you confirm my understanding that #180 needs to be completed before this can be implemented? In order to add notes for **each child**, need details regarding children must be entered individually.

> A user creating a need has the ability to add notes for **each child** included in the request.

**Comment by haseebelahi** [comment]
Ok, I can work on #180 as well if that needs to be done before this!

**Comment by sarahwicks** [comment]
@haseebelahi, @benjaminwood - Yes, this is blocked by #180. I updated the ticket and indicated the dependencies on both tickets. Thank you! cc: @SarahOMD @DarcyBlank
